src/main/resources/OnlineApplication.xsd

```diff
@@ -2684,6 +2684,27 @@
 			</xsd:element>
 		</xsd:sequence>
 	</xsd:complexType>
+	<xsd:element name="AuthorizedRepresentative" >
+		<xsd:complexType>
+			<xsd:sequence>
+				<xsd:element name="LegalGuardianInd" type="xsd:boolean" minOccurs="0">
+					<xsd:annotation>
+						<xsd:documentation>F# 82</xsd:documentation>
+					</xsd:annotation>
+				</xsd:element>
+				<xsd:element name="SomeoneActAsRepresentativeInd" type="xsd:boolean" minOccurs="0">
+					<xsd:annotation>
+						<xsd:documentation>F# 82</xsd:documentation>
+					</xsd:annotation>
+				</xsd:element>
+				<xsd:element name="Representative" type="model:RepresentativeType" minOccurs="0">
+					<xsd:annotation>
+						<xsd:documentation>F# 82 - 96</xsd:documentation>
+					</xsd:annotation>
+				</xsd:element>
+			</xsd:sequence>
+		</xsd:complexType>
+	</xsd:element>

```

Comment:
This didn't need to be added. It's already on line 744.

https://github.com/codeforamerica/shiba/blob/0a1bd1668d3f2643cebc1029e091bb28cb758b15/src/main/resources/OnlineApplication.xsd#L744-L764

Reply:
Thanks for that catch!